backoff/backoff.go

```diff
@@ -39,5 +39,5 @@ func jitter(n int64) int64 {
 		return 0
 	}
 
-	return n/2 + rand.Int63n(n/2)
```

Comment:
Even when `math.rand` and `crypto/rand` are [getting closer together](https://go.dev/blog/chacha8rand), we could also just use `crypto/rand` here.

Reply:
It needs way more code changes with `crypto/rand` than just replacing it like `error` handling, `bigInt` usage etc, and since we don't actually need cryptographically secure randomness here, I chose to just suppress it.